docs/sources/clients/promtail/installation.md

```diff
@@ -78,6 +78,11 @@ spec:
         image: grafana/promtail
         args:
         - -config.file=/etc/promtail/promtail.yaml
+        env: 
+        - name: 'HOSTNAME', # needed when using kubernetes_sd_configs
```

Comment:
Thanks @robbymilo :).

I can confirm we use `HOSTNAME` on [our cloud promtail config](https://github.com/grafana/loki/blob/main/tools/promtail.sh#L281-L284).